best-practices/README.md

```diff
@@ -104,11 +104,18 @@ Rails
 Testing
 -------
 
+(See also the rspec style guide that appears in this section)
+
+* First #describe what you are doing.
+* Then establish the #context.
+* #it only expects one thing (within reason).
```

Comment:
This & the argument about nesting above both seem to be closely related to slow tests with lots of setup - which I think we all agree we want to avoid in the future?